src/api/db/migrate/20170323145236_add_user_id_to_bs_request.rb

```diff
@@ -0,0 +1,21 @@
+class AddUserIdToBsRequest < ActiveRecord::Migration[5.0]
+  def up
+    add_reference(:bs_requests, :user)
+    User.all.pluck(:id, :login,).each do |user|
+      execute "UPDATE bs_requests SET user_id = '#{user[0]}' WHERE creator = '#{user[1]}'"
+    end
```

Comment:
and I would do a transaction around. To avoid the situation of half migrated data in case of error.